charts/kubescape-operator/assets/host-scanner-definition.yaml

```diff
@@ -73,12 +73,7 @@ spec:
             containerPort: 7888
             protocol: TCP
         resources:
-          limits:
-            cpu: 0.4m
-            memory: 400Mi
-          requests:
-            cpu: 0.1m
-            memory: 200Mi
+{{ toYaml .Values.hostScanner.resources | indent 10 }}
```

Comment:
I am not sure this daemonset is used anymore... if it was we should have seen the updated in unittest

Reply:
sorry my bad